### DISCUSSION TITLE
Shortcuts for Highlighting

### DISCUSSION THREAD

**Discussion by Dgram19** [sumatrapdfreader/sumatrapdf#4344] [Ideas]
Hi, first of all, thank you for the work you do. 
When I go to highlight I use the shortcut "ctrl+a", and this allows me to highlight in yellow. It would be nice if it would be possible to add more shortcuts for highlighting, for example "ctrl+a" --> yellow, "ctrl + b" --> green and so on.
Thank you

**Comment by kjk** [comment]
In pre-release 16374 or later (https://www.sumatrapdfreader.org/prerelease) you can do:

```
Shortcuts [
	[
		Cmd = CmdCreateAnnotHighlight #00ff00   
		Key = b
	]
]
```
This creates shortcut `b` for creating green highlight
See https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

**Reply by Dgram19** [reply]
Thank you
